orcid-core/src/main/java/org/orcid/core/oauth/service/OrcidTokenStoreServiceImpl.java

```diff
@@ -132,7 +133,7 @@ public OAuth2AccessToken readAccessToken(String tokenValue) {
      *            The token to remove from the database.
      */
     @Override
-    @Transactional
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Hey @wjrsimpson could you please take a look at this one? So, with Propagation.REQUIRES_NEW we create a separate transaction to persist the disable token query.